### DISCUSSION TITLE
[Feature] Go to timeline view from any photo or video

### DISCUSSION THREAD

**Discussion by mrupnikm** [immich-app/immich#7989] [Feature Request]
### The feature

Love your work!

I am missing a way to get to the timeline view of the photo or video that I have selected. Maybe this would be in the form of right click or it would be a button/view when viewing the full screen of a photo.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4115
